example/ios/Podfile

```diff
@@ -3,11 +3,17 @@ require_relative '../node_modules/@react-native-community/cli-platform-ios/nativ
 
 platform :ios, '15.0'
 use_frameworks! :linkage => :static
+$RN_FABRIC_ENABLED = true
+$RN_TURBOMODULE_ENABLED = true
 
 target 'example' do
   config = use_native_modules!
 
-  use_react_native!(:path => config[:reactNativePath])
+  use_react_native!(
+    :path => config[:reactNativePath],
+    :fabric_enabled => $RN_FABRIC_ENABLED,
+    :hermes_enabled => true
+  )
 
```

Comment:
We now need to use 'Fabric' to get iOS to build with the RN frameworks. Added hermes, as I believe that's necessary for fabric. Annoying.